gem/spec/fixtures/carthage_after/AutoEarlGrey/EarlGrey.swift

```diff
@@ -0,0 +1,78 @@
+//
```

Comment:
this should be re-synced from https://github.com/google/EarlGrey/blob/master/Demo/EarlGreyExample/EarlGreyExampleSwiftTests/EarlGrey.swift